partiql-lang/build.gradle.kts

```diff
@@ -64,6 +64,24 @@ dependencies {
     }
 }
 
+val relocations = mapOf(
+    "org.antlr" to "org.partiql.thirdparty.antlr"
+)
+
+tasks.shadowJar {
+    configurations = listOf(project.configurations.shadow.get())
+    for ((from, to) in relocations) {
+        relocate(from, to)
+    }
+}
+
+// Workaround for https://github.com/johnrengelman/shadow/issues/651
+components.withType(AdhocComponentWithVariants::class.java).forEach { c ->
+    c.withVariantsFromConfiguration(project.configurations.shadowRuntimeElements.get()) {
+        skip()
+    }
+}
+
```

Comment:
Why does partiql-lang have an ANTLR runtime dependency to begin with? Same with the CSV dependency (that's a CLI concern no?). I'm wondering if these deps were mistakenly not removed during package split.

Reply:
Discussed offline. Adding the notes here:
- https://github.com/partiql/partiql-lang-kotlin/blob/shade-antlr/partiql-lang/src/main/kotlin/org/partiql/lang/eval/io/DelimitedValues.kt#L73-L105 is where the CSV dependency is used
- https://github.com/partiql/partiql-lang-kotlin/blob/shade-antlr/partiql-lang/src/main/kotlin/org/partiql/lang/syntax/impl/PartiQLPigParser.kt uses the ANTLR runtime